src/modules/Experiences/AddOrEditExperienceForm/AddOrEditExperienceForm.js

```diff
@@ -133,6 +139,11 @@ const AddOrEditExperienceForm = ({ experience, onSubmit, handleCancel, classes }
   if (typeof values.start_year === String) {
     values.start_year = null;
   }
+
+  if (experience.type === 'Accomplishment') {
+    values.end_month = 'none';
+    values.end_year = 0;
+  }
```

Comment:
This feels like it should happen in handleFormSubmit, and also like it should happen in general whenever values.end_month and values.end_year are null? It seems like in general we need a cleaner way of handling end_month and end_year in this module

Reply:
It occurs to me that fixing up how end_month and end_year work isn't super related to the PR, though, so it probably shouldn't block the merge of this specific thing

Reply:
Created isEndDateNull() in helper.js to handle any (end_month/year === null) situation